src/components/atoms/AtomIcon.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <img class="imageBorder" :src="src" />
+</template>
+
+<script lang="ts" setup>
+defineProps({
+  src: {
+    type: String,
+    required: true,
+  },
+}
+);
```

Comment:
Code-Styling: Remove line break.
```suggestion
});
```